File: 123. CSC102 Quiz1 Mock up 2020: Matching/123.py
def main():
    input_str = input()
    s1, s2 = input_str.split()

    if s1 == s2:
        print("Perfect Match")
    elif s1.lower() == s2.lower() or s1.lower() in s2.lower() or s2.lower() in s1.lower():
        print("Partial Match")
    else:
        print("Not Match")

if __name__ == "__main__":
    main()

File: 123. CSC102 Quiz1 Mock up 2020: Matching/123.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s1 = sc.next();
		String s2 = sc.next();

		if (s1.equals(s2)) {
			System.out.println("Perfect Match");
		} else if (s1.equalsIgnoreCase(s2)
               || s1.toLowerCase().contains(s2.toLowerCase())
               || s2.toLowerCase().contains(s1.toLowerCase())) {
			System.out.println("Partial Match");
		} else {
			System.out.println("Not Match");
		}
	}
}
